sdk/storage/azure-storage-files-datalake/inc/azure/storage/files/datalake/datalake_responses.hpp

```diff
@@ -443,6 +451,32 @@ namespace Azure { namespace Storage { namespace Files { namespace DataLake {
 
     using SetPathPermissionsResult = SetPathAccessControlListResult;
 
+    /**
+     * @brief A path that has been soft deleted.
+     */
+    struct PathDeletedItem final
+    {
+      /**
+       * The name of the path.
+       */
+      std::string Name;
+
+      /**
+       * The deletion ID associated with the deleted path.
+       */
+      std::string DeletionId;
+
+      /**
+       * When the path was deleted.
+       */
+      DateTime DeletedOn;
+
+      /**
+       * The number of days left before the soft deleted path will be permanently deleted.
+       */
+      int64_t RemainingRetentionDays;
```

Comment:
Some compilers give warnings for uninitialized primitive types. But it's fine if CI passes.